utils/test/jest.setup.js

```diff
@@ -15,3 +15,6 @@ import 'jest-styled-components';
 import Adapter from 'enzyme-adapter-react-16';
 import Enzyme from 'enzyme';
 Enzyme.configure({ adapter: new Adapter() });
+
+// Enable async/await
+import '@babel/polyfill';
```

Comment:
Do we need this if we're running in a node version that supports it natively?

Reply:
From what I could tell, yes 😭 The jest environment is slightly different

Reply:
Would continually run into `regeneratorRuntime` errors